src/main/features/response/form/models/statement-of-means/monthlyIncome.ts

```diff
@@ -1,89 +1,90 @@
 import { ValidateIf, ValidateNested } from 'class-validator'
 
-import { IncomeExpenseSource } from './incomeExpenseSource'
+import { IncomeSource } from './incomeSource'
 
-export class SourceNames {
-  static readonly SALARY = 'Income from your job'
+export class FieldNames {
+  static readonly SALARY = 'income'
   static readonly UNIVERSAL_CREDIT = 'Universal Credit'
-  static readonly JOBSEEKER_ALLOWANCE_INCOME = 'Jobseeker’s Allowance (income based)'
-  static readonly JOBSEEKER_ALLOWANCE_CONTRIBUTION = 'Jobseeker’s Allowance (contribution based)'
+  static readonly JOBSEEKER_ALLOWANCE_INCOME = 'income-based Jobseeker’s Allowance'
+  static readonly JOBSEEKER_ALLOWANCE_CONTRIBUTION = 'contribution-based Jobseeker’s Allowance'
   static readonly INCOME_SUPPORT = 'Income Support'
   static readonly WORKING_TAX_CREDIT = 'Working Tax Credit'
   static readonly CHILD_TAX_CREDIT = 'Child Tax Credit'
   static readonly CHILD_BENEFIT = 'Child Benefit'
   static readonly COUNCIL_TAX_SUPPORT = 'Council Tax Support'
-  static readonly PENSION = 'Pension (paid to you)'
+  static readonly PENSION = 'pension payments'
+  static readonly OTHER = 'other expense'
```

Comment:
is this used?

Reply:
Indeed that was just left behind by mistake - good spot!